test/DEFAULT_FASTRTPS_PROFILES.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I am missing a FastRTPS XML for sync when 2 processes. It'd consist in the same, eliminating the `<library_settings>` tag. Since you're loading this files with `FASTRTPS_DEFAULT_PROFILES_FILE`, the files can take any name.

Reply:
https://github.com/ros2/buildfarm_perf_tests/pull/26/commits/2c7c2bada70f7e726234ca9e6421e86dbbfc7d6e

Reply:
:+1:

Reply:
Are there any undesired behaviors if we use this config for all of the synchronous Fast-RTPS tests, including the two-process ones? It would be nice to have only one profile XML if possible.

Reply:
Yes, most definitely. With this file, communication is limited to participants in the same process